src/common/components/PageHeader/PageHeader.tsx

```diff
@@ -13,42 +18,95 @@ export interface PageHeaderProps {
   title: string
   routes?: TMultiToggle
   sortByOptions?: PageHeaderSortByOptions[]
+  variant?: string
+  breadcrumbs?: TBreadcrumbProps[]
+  sortByText?: string
+  sortByTextClassName?: string
 }
 
 /**
  * This is a presentational component, meaning it holds no state.
  * State should be handled by parent component.
  */
 const PageHeader = (props: PageHeaderProps): JSX.Element => {
-  const { title, routes, sortByOptions } = props
+  const {
+    title,
+    routes,
+    sortByOptions,
+    breadcrumbs,
+    variant,
+    sortByText,
+    sortByTextClassName = '',
+  } = props
+
+  const renderBreadcrumbs = () => {
+    if (!breadcrumbs) {
+      return null
+    }
+
+    return (
+      <div className='wrapper py-0 flex font-normal text-14px leading-19px text-gray-71 mb-12px pt-12px'>
+        {breadcrumbs.map((breadcrumb, index) => {
+          if (breadcrumb.route) {
+            return (
+              <React.Fragment key={index}>
+                <a href={breadcrumb.route}>{breadcrumb.label}</a>
+                <span className='px-6px'>/</span>
+              </React.Fragment>
+            )
+          }
+
+          return <span key={index}>{breadcrumb.label}</span>
+        })}
+      </div>
+    )
+  }
 
   return (
-    <div className='bg-white text-gray-1a'>
-      <div className='wrapper py-30px'>
-        <div className='flex justify-between'>
-          <div className='flex items-center'>
-            <h1 className='pr-70px font-semibold text-gray-23'>{title}</h1>
-            {routes && <MultiToggleSwitch {...routes} />}
-          </div>
-          {sortByOptions?.length && (
+    <>
+      {renderBreadcrumbs()}
```

Comment:
Please don't use render functions (in general), such code can be easily moved to a separate component, this component would become shorter and faster to read and maintain.

I see that breadcrumbs were already moved to a separate component in other PR, so this can be resolved during merging

Reply:
Ok @CrazyOptimist